src/data/moves_info.h

```diff
@@ -19077,7 +19077,7 @@ const struct MoveInfo gMovesInfo[MOVES_COUNT_DYNAMAX] =
 
     [MOVE_MAKE_IT_RAIN] =
     {
-        .name = COMPOUND_STRING("Make It Rain"),
+        .name = COMPOUND_STRING("MAKE IT RAIN"),
```

Comment:
With the changes made to prevent decapping KO, TM, EV, etc., wouldn't this decap to Make IT Rain? Same with Abilities like Sword OF Ruin